.github/workflows/release-trigger.yaml

```diff
@@ -0,0 +1,26 @@
+# This Github action is the release trigger for Azure/karpenter-provider-azure
+# It will auto-generate the next semantic release version and tag the git ref
+
+name: Release Trigger
+on:
+    workflow_dispatch:
+
+jobs:
+  generate-sem-ver:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@17d0e2bd7d51742c71671bd19fa12bdc9d40a3d6 # v2.8.1
+        with:
+          egress-policy: audit
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20.x' # semantic-release requires Node version 20.8.1 or higher
+      - name: semantic-release
+        run: |
+          npx semantic-release@18
```

Comment:
Curios about where this version comes from, and how we will be updating it?
(Also wondering if there is a reason this is not available as GH action?)

Reply:
@18 is the most recent major version for `semantic-release`, and the recommended approach they have for using it.

As for no action, I also was wondering the same thing. From my exploration thus far I still haven't found a great answer to it, but I'm thinking that it might be simply due to the complexity of it, that they've decided not to publish an actual action.

Reply:
Ensuring a procedure for updating this dependency is an important problem

Reply:
Resolving to unblock completion, but updating both this version and node is something that will need a solution.